gcloud-java-pubsub/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Client for Pub/Sub
```

Comment:
Not all of the sections are relevant yet - there isn't an example application and there isn't documentation on testing to point to. Do you want me to include the remainder of the sections?